From 1f4dec2783ec02031a2e24c1820fb308f6370c57 Mon Sep 17 00:00:00 2001
From: wjyrich <1071633242@qq.com>
Date: Thu, 8 Aug 2024 14:27:23 +0800
Subject: [PATCH] install

---
 qprogram-starter/qprogram-starter.pro | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/qprogram-starter/qprogram-starter.pro b/qprogram-starter/qprogram-starter.pro
index 6593349..9ea3e9c 100644
--- a/qprogram-starter/qprogram-starter.pro
+++ b/qprogram-starter/qprogram-starter.pro
@@ -39,13 +39,13 @@ TRANSLATIONS = src/translations/qprogram-starter-ast.ts\
                src/translations/qprogram-starter-uk.ts
 
 # install
-target.path = /usr/bin
+target.path = $$PREFIX/bin
 translation.files = src/translations/*.qm
-translation.path = /usr/share/qprogram-starter/translations
+translation.path = $$PREFIX/share/qprogram-starter/translations
 autostart.files = autostart/qprogram-starter.desktop
-autostart.path = /usr/share/applications
+autostart.path = $$PREFIX/share/applications
 icon.files = src/pixmap/qprogram-starter.xpm
-icon.path = /usr/share/icons/hicolor/32x32/apps
+icon.path = $$PREFIX/share/icons/hicolor/32x32/apps
 
 INSTALLS = target translation autostart icon
 
-- 
2.43.4

